client/src/testFixtures/java/org/apache/cassandra/sidecar/client/SidecarClientTest.java

```diff
@@ -1622,6 +1624,42 @@ public void onError(Throwable throwable)
         assertThat(new String(baos.toByteArray(), StandardCharsets.UTF_8)).isEqualTo("Test Content");
     }
 
+    @Test
+    public void testStreamsStats() throws Exception
+    {
+        String streamStatsResponseAsString = "{\"operationMode\":\"NORMAL\"," +
+                                             "\"streamsProgressStats\":{\"totalFilesToReceive\":7," +
+                                             "\"totalFilesReceived\":7,\"totalBytesToReceive\":15088," +
+                                             "\"totalBytesReceived\":15088,\"totalFilesToSend\":2,\"totalFilesSent\":2," +
+                                             "\"totalBytesToSend\":1024,\"totalBytesSent\":1024}}";
+
+        MockResponse response = new MockResponse().setResponseCode(OK.code()).setBody(streamStatsResponseAsString);
+        enqueue(response);
+
+        for (MockWebServer server : servers)
+        {
+            SidecarInstanceImpl sidecarInstance = RequestExecutorTest.newSidecarInstance(server);
+            StreamStatsResponse result = client.streamsStats(sidecarInstance).get(30, TimeUnit.SECONDS);
+            assertThat(result).isNotNull();
+            assertThat(result.operationMode()).isNotNull().isEqualTo("NORMAL");
+            StreamsProgressStats progressStats = result.streamsProgressStats();
+            assertThat(progressStats).isNotNull();
+            assertThat(progressStats.totalFilesToSend()).isNotNull()
+                                                        .isEqualTo(progressStats.totalFilesSent())
+                                                        .isEqualTo(2);
+            assertThat(progressStats.totalBytesToSend()).isNotNull()
+                                                      .isEqualTo(progressStats.totalBytesSent())
+                                                      .isEqualTo(1024);
+            assertThat(progressStats.totalBytesToReceive()).isNotNull()
+                                                           .isEqualTo(progressStats.totalBytesReceived())
+                                                           .isEqualTo(15088);
+            assertThat(progressStats.totalFilesToReceive()).isNotNull()
+                                                           .isEqualTo(progressStats.totalFilesReceived())
+                                                           .isEqualTo(7);
```

Comment:
nit: create `StreamsProgressStats` object and serialize it into string, instead of declaring  `streamStatsResponseAsString`. For assertion, now it can check the received `StreamsProgressStats` has the same values as the original `StreamsProgressStats`